.github/workflows/format.yml

```diff
@@ -23,6 +26,10 @@ jobs:
       with:
         # Slow hooks are marked with manual - slow is okay here, run them too
         extra_args: --hook-stage manual --all-files
+    - name: PyLint
```

Comment:
Just add pylint as another pre-commit hook, it has native support.

Reply:
Pylint usually requires it be installed into the same environment that your package is installed into. Except in this case, where local runs are actually just fine. 🤦 It's also often slower, but again, not really since this is small.

Reply:
Ah nvm, it has the same problem as flake8, mypy, and other checks. :/